cmds/core/ls/ls_plan9.go

```diff
@@ -18,18 +18,18 @@ import (
 
 var final = flag.BoolP("print-last", "p", false, "Print only the final path element of each file name")
 
-func printFile(w io.Writer, stringer ls.Stringer, f file) {
+func (c cmd) printFile(w io.Writer, stringer ls.Stringer, f file) {
 	if f.err != nil {
 		fmt.Fprintln(w, f.err)
 		return
 	}
 	// Hide .files unless -a was given
-	if *all || !strings.HasPrefix(f.lsfi.Name, ".") {
+	if c.all || !strings.HasPrefix(f.lsfi.Name, ".") {
 		// Unless they said -p, we always print the full path
 		if !*final {
```

Comment:
The last global, but it should not be a problem for tamago.

Reply:
yeah, I don't know how long I care about Plan 9, but we'll see. We might actually want this option for the standard ls anyway ...